Cornelica/scenario/014_02b.txt

```diff
@@ -4,9 +4,9 @@
 @huki b=3 id=3
 @move_f f=アルネ@笑み in=c t=150
 
-[Arne] "Welcome back! Mphm... you really like to show off!"
+[Arne]"Welcome back! Mphm... you really like to show off!"
```

Comment:
I think `Mmph` looks better for that.

Reply:
I hate those effing sound, i'm going crazy...